WegliKit/Sources/ImagesFeature/ImagesCore.swift

```diff
@@ -131,7 +143,40 @@ public let imagesReducer = Reducer<ImagesViewState, ImagesViewAction, ImagesView
     default:
       return .none
     }
-    
+
+  case .takePhotosButtonTapped:
+    switch env.cameraAccessClient.authorizationStatus() {
+    case .authorized:
+      return Effect(value: .setShowCamera(true))
+    case .denied:
+      state.alert = .init(title: TextState(L10n.Camera.Alert.accessDenied))
+      return .none
+    case .notDetermined:
+      return Effect(value: .requestCameraAccess)
+    case .restricted:
+      // TODO: How to handle this?
```

Comment:
From the UI we could maybe disable the button but its also ok to do nothing I guess since I think the apps users have unrestricted access. WDYT?

Reply:
Oh yeah I forgot to work on this TODO. 
I agree with you, just doing nothing would be ok in this case.